Modules/Setup

```diff
@@ -218,7 +218,7 @@ time timemodule.c
 #_dbm _dbmmodule.c -lgdbm_compat -DUSE_GDBM_COMPAT
 #_gdbm _gdbmmodule.c -lgdbm
 #_lzma _lzmamodule.c -llzma
-#_sqlite3 _sqlite/connection.c _sqlite/cursor.c _sqlite/microprotocols.c _sqlite/module.c _sqlite/prepare_protocol.c _sqlite/row.c _sqlite/statement.c _sqlite/util.c -lsqlite3
+#_sqlite3 _sqlite/connection.c _sqlite/cursor.c _sqlite/microprotocols.c _sqlite/module.c _sqlite/prepare_protocol.c _sqlite/row.c _sqlite/statement.c _sqlite/util.c $(PY_SQLITE_LDFLAGS)
```

Comment:
Please include CFLAGS here, too.

Reply:
Hm, I thought I already did, but I guess that bit disappeared in a `git reset` somewhere before last commit :) Will fix.